src/manage_nsfs/nc_lifecycle.js

```diff
@@ -161,7 +161,11 @@ async function process_buckets(config_fs, bucket_names, system_json) {
  */
 async function process_bucket(config_fs, bucket_name, system_json) {
     const bucket_json = await config_fs.get_bucket_by_name(bucket_name, config_fs_options);
-    const account = { email: '', nsfs_account_config: config_fs.fs_context, access_keys: [] };
+    const account = await config_fs.get_identity_by_id(bucket_json.owner_account, undefined, {silent_if_missing: true});
```

Comment:
type should be account so it'll be backwards compatible

Reply:
changed the type to account